src/flag_gems/ops/vdot.py

```diff
@@ -0,0 +1,206 @@
+import logging
+
+import torch
+import triton
+import triton.language as tl
+from torch import Tensor
+
+from .. import runtime
+from ..utils import libentry
+
+tl_support_split = hasattr(tl, "split")
```

Comment:
@Galaxy1458 could we add a check for these intrinsics supported only in triton v3?